src/System.Collections/src/System/Collections/Generic/HashSet.cs

```diff
@@ -131,6 +135,21 @@ public HashSet(IEnumerable<T> collection, IEqualityComparer<T> comparer)
             }
         }
 
+        public HashSet(int capacity, IEqualityComparer<T> comparer)
+            : this(comparer)
+        {
+            if (capacity < 0)
+            {
+                throw new ArgumentOutOfRangeException("capacity");
+            }
+            Contract.EndContractBlock();
```

Comment:
I wonder (not tied to this commit) if we should strip Contract.EndContractBlock et al since we're not using the code contract tooling.